CalibPPS/ESProducers/plugins/PPSAlignmentConfigESSource.cc

```diff
@@ -1,8 +1,6 @@
 /****************************************************************************
  *
- *  CalibPPS/ESProducers/plugins/PPSAlignmentConfigESSource.cc
- * 
- *  Description: Constructs PPSAlignmentConfig instance
+ *  This module is obsolete! Use PPSAlignmentConfigurationESSource instead.

```

Comment:
Should there be a real exception thrown if somebody wants to use this? I'm not sure a single comment in the code is sufficient to stop people to use it haha

Reply:
I just wanted to make sure no one is confused when they see two very similar classes in one directory, but throwing an exception is a good idea too. I'll add it.

Reply:
if the module is obsolete, what's the reason to keep it in the code-base? 
Are there still clients of it?

Reply:
`PPSAlignmentConfiguration` has fully replaced `PPSAlignmentConfig`, so nobody will use its ESSource in the future. To be honest, I wanted to delete all the `PPSAlignmentConfig` auxiliary classes in one of the previous PRs (https://github.com/cms-sw/cmssw/pull/35174#discussion_r704723640), but eventually we decided to keep them. Do you suggest to delete them now?

Reply:
the less code is in the stack, the less maintenance it will require. If the functionality is fully replaced by other modules, deleting the obsolete files sounds like a good option (and will also automatically enforce that nobody uses them by mistake).

Reply:
Ok then, I'll delete them. Am I right though that I can't delete the `PPSAlignmentConfig` class? It has a few payloads in the DB.

Reply:
>  Am I right though that I can't delete the PPSAlignmentConfig class? It has a few payloads in the DB.

provided they are not included in any `autoCond` GlobalTag you technically *could* delete the C++ class as well, though if I am not mistaken AlCa/DB discourages that practice @cms-sw/alca-l2 can comment further.

Reply:
Ok, thanks! I will leave the class for now.